Resources/Prototypes/Entities/Objects/Fun/instruments.yml

```diff
@@ -167,6 +167,11 @@
   - type: Item
     size: 24
     sprite: Objects/Fun/Instruments/saxophone.rsi
+  - type: Extractable
+    juiceSolution:
+      reagents:
+        - ReagentId: Saxoite
+          Quantity: 10
```

Comment:
this is a little silly buuuuuuuuut part of me doesn't really care

Reply:
i had to include a jazz joke
(plus this locks it behind two separate hard-to-obtain items, which is good imo, makes this showing up rarer)